controllers/fetchers/url_fetcher.go

```diff
@@ -30,7 +31,8 @@ func FetchDashboardFromUrl(dashboard *v1beta1.GrafanaDashboard) ([]byte, error)
 		return nil, err
 	}
 
-	client := client2.NewInstrumentedRoundTripper(fmt.Sprintf("%v/%v", dashboard.Namespace, dashboard.Name), metrics.DashboardUrlRequests, true)
+	// tlsConfig := client.InsecureTLSConfiguration
```

Comment:
```suggestion
```

Reply:
Oups, my bad. Normally, it should be fine